tests/utils.js

```diff
@@ -499,30 +518,47 @@ module.exports = {
   /**
    * Update settings and refresh if required
    *
-   * @param      {Object}   updates  Object containing all updates you wish to
-   *                                 make
-   * @param      {Boolean}  ignoreRefresh  don't bother refreshing
-   * @return     {Promise}  completion promise
+   * @param {Object}         updates  Object containing all updates you wish to
+   *                                  make
+   * @param  {Boolean|String} ignoreReload if false, will wait for reload modal and reload. if truthy, will tail
```

Comment:
I was confused when I saw this being called with a boolean then a string so having this annotation helps.

Reply:
Yea, I've been toying around with the idea of making this API different. 
One idea was to always need to pass a string to target a service, but that would require lots of edits where this method is called. 
I'm not sure what is best, so I went with the approach that required least edits.